docs/whitepapers/shortpaper.mdx

```diff
@@ -0,0 +1,641 @@
+---
+id: shortpaper
+title: Shortpaper (DRAFT)
+sidebar_label: Shortpaper (DRAFT)
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import TOCInline from '@theme/TOCInline';
+
+--TODO: equations are being duplicated for unknown reasons. Also unable to add stylesheets for equations as outlined in https://docusaurus.io/docs/markdown-features/math-equations to the config file. This should be solved.
+
+
+<a href={useBaseUrl("/whitepapers/economic/shortpaper_latest.pdf")} target="_blank">Download Shortpaper</a>
```

Comment:
has this file been generated yet? is it supposed to be in this PR?

Reply:
I'm going to remove this link. Better people don't download hard copies yet as the document isn't complete.